src/lib/slack/commands/SlashCommand.js

```diff
@@ -26,26 +26,27 @@ const SlashCommand = {
     },
 
     sendResponse: async function(block) {
-        await request({
-	        method: 'POST',
-	        uri: this.responseUrl,
-	        json: true,
-	        body: block
-        }).catch((err) => {
+        try {
+            await request({
+                method: 'POST',
+                uri: this.responseUrl,
+                json: true,
+                body: block
+            })
+        } catch {
             console.error('ERROR ** Error posting to Slack response URL for slash command', err);
-        });
+        }
     }
-}
+};
 
 function requireParams(body, params) {
-    var validated = true;
-    for (var param of params) {
+    for (const param of params) {
         if (!body[param]) {
             console.error(`ERROR ** Unable to init SlashCommand, no ${param} param found in body:`, body);
-            validated = false;
+            return false;
```

Comment:
Nice, for some reason I thought `return` did not break `for` loops.

Reply:
That could be true, I can do some validation there. I can't say the my JS is that great